lib/components/src/blocks/ArgsTable/ArgRow.tsx

```diff
@@ -69,28 +70,32 @@ export const ArgRow: FC<ArgRowProps> = (props) => {
         <Name>{name}</Name>
         {required ? <Required title="Required">*</Required> : null}
       </td>
-      <td>
-        {hasDescription && (
-          <Description>
-            <Markdown>{description}</Markdown>
-          </Description>
-        )}
-        {table.jsDocTags != null ? (
-          <>
-            <TypeWithJsDoc hasDescription={hasDescription}>
+      {compact ? null : (
```

Comment:
what about instead of `compact ? null : (something)` you have a const like `isExpanded = !compact` and do 
`isExpanded && (something)`?

Reply:
does `{false}` work in JSX?

Reply:
yes, it should just not render anything.
https://codepen.io/yannbf/pen/LYpMdmr